packages/icon/src/react/icons/index.ts

```diff
@@ -72,6 +72,7 @@ export { HistoryIcon } from './History.dist'
 export { HomeAdminToolsIcon } from './HomeAdminTools.dist'
 export { HomeContentToolsIcon } from './HomeContentTools.dist'
 export { HomeIcon } from './Home.dist'
+export { IlxIcon } from './Ilx.dist'
```

Comment:
You don't need to do this - it is auto-generated. I thought I put a comment on that file? 🤔

Reply:
I didn't see the comment, but I figured there was some automation there and found it afterwards.